src/templates/ApiEndpoint.tsx

```diff
@@ -640,8 +637,6 @@ export default function ApiEndpoint({ data }: { data: ApiEndpointData }): JSX.El
                         </div>
                     )}
 
-                    <ReactMarkdown>{items[0].operationSpec?.description}</ReactMarkdown>
```

Comment:
This never rendered quite right and is actually not a useful description in most cases